src/examples/2_moves_introduction.ipynb

```diff
@@ -418,9 +418,9 @@
     "    time.sleep(0.5)\n",
     "\n",
     "# Write a sequence for the left arm\n",
-    "reachy.l_arm.goto_joints([0, 0, 0, -90, 0, 0, 0], duration = 3)  # id=1\n",
-    "reachy.l_arm.goto_joints([0, 0, 0, 0, 0, 0, 0], duration = 2)  # id=2\n",
-    "reachy.l_arm.goto_joints([0, 0, 0, -90, 0, 0, 0], duration = 3)  # id=3\n",
+    "reachy.l_arm.goto_joints([0, 10, 15, -90, 0, 0, 0], duration = 3)  # id=1\n",
+    "reachy.l_arm.goto_joints([0, 10, 15, 0, 0, 0, 0], duration = 2)  # id=2\n",
```

Comment:
I would set 10,15, **-30** (instead of 0) to be sure that the arm doesn't touch the structure